src/Raven.Server/Documents/DocumentFlags.cs

```diff
@@ -18,5 +18,6 @@ public enum DocumentFlags
         Reserved3 = 0x80,
 
         HasAttachments = 0x100,
+        Revision = 0x200
```

Comment:
Is there a way not to have it here? This is legacy value.
If not then maybe we can rename it to LegacyRevision? What about using Versioned?